android/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java

```diff
@@ -37,7 +37,29 @@ private static boolean isRunningOnGenymotion() {
     }
 
     private static boolean isRunningOnStockEmulator() {
```

Comment:
This code is synced in and not authored by us. Manual changes to this file will be erased.